docs/getting-started/installation.md

```diff
@@ -25,9 +25,9 @@ wget https://raw.githubusercontent.com/dokku/dokku/v0.10.4/bootstrap.sh;
 sudo DOKKU_TAG=v0.10.4 bash bootstrap.sh
 ```
 
-The installation process takes about 5-10 minutes, depending upon internet connection speed. 
+The installation process takes about 5-10 minutes, depending upon internet connection speed.
 
-If you're using Debian 8 or Ubuntu 14.04, make sure your package manager is configured to install a sufficiently recent version of nginx<sup>[3]</sup>, otherwise, the installation may fail due to "unmet dependencies" relating nginx.
+If you're using Debian 8 or Ubuntu 14.04, make sure your package manager is configured to install a sufficiently recent version of nginx<sup>[3]</sup>, otherwise, the installation may fail due to "unmet dependencies" relating nginx. For [HTTP/2 support](configuration/ssl.md#http2-support), nginx >= 1.11.5<sup>[4]</sup> is required.
```

Comment:
Actually, I'm not sure how this anchor link will look like. The heading is `HTTP/2 support`, is `/` removed completely or replaced by `-`?